README.md

```diff
@@ -28,8 +28,7 @@
 
 ## Prerequisites
 
-- :gear: [NodeJS](https://nodejs.org/) (LTS)
-- :toolbox: [Yarn](https://yarnpkg.com/)
+First follow the prerequisites list [here](./CONTRIBUTING.md#prerequisites)
```

Comment:
I am not a fan of referencing the contribution.md here. We have references to that already in the readme and contributors should look at that anyways. The readme is more of a quickstart to get Lodestar to run and for most operating systems (Linux & WSL) it will just work.

Let's get some more opinions on this

Reply:
Makes sense, I can revert this. The idea was to not duplicate now your point makes sense.
I would expect the build to fail for mac users with mac silicon due to missing `python` tough.

Reply:
I tend to agree that this should not be referenced here.  Those two docs are for different purposes and have different audiences